src/pages/faq.astro

```diff
@@ -21,7 +22,9 @@ const faq = await getSortedFaq();
                     <details>
                         <summary>{f.data.title}</summary>
 
-                        <Content />
+                        <div class="faq-content">
```

Comment:
why does this need a wrapper div? Just use `details :global(p)` instead

Reply:
was trying to avoid adding more global styles than necessary

Reply:
details :global(p) is already scoped because the `details` selector is not global